contrib/checkstyle/src/com/etsy/sbt/checkstyle/CheckstyleXSLTSettings.scala

```diff
@@ -0,0 +1,10 @@
+package com.etsy.sbt.checkstyle
+
+import java.io.File
+
+/**
+ * Configuration for a single XSLT transformation of the checkstyle output file
+ *
+ * @author Andrew Johnson
+ */
+case class CheckstyleXSLTSettings(xslt: File, output: File)
```

Comment:
Do we actually need an `output: File` for each XSLT transformation? If we're always applying the same transformations in the same sequence on the input checkstyle config between using it, the choice of output file name seems irrelevant and seems like something Mill can manage

Reply:
I think the idea is that each transformation has its own output file (e.g. one transformation for a human-readable HTML page and one for some kind of internal XML report). To be honest it's not a feature I've used, just trying to retain parity with the SBT plugin.